src/random/cuda/RngStatePointers.hh

```diff
@@ -7,6 +7,14 @@
 //---------------------------------------------------------------------------//
 #pragma once
 
+/*!
+ * \def QUALIFIERS
+ *
+ * Define NVCC QUALIFIERS so as to make curand functions work on both host
+ * and device.
```

Comment:
Can you add a brief explanation that this is modifying an undocumented `curand_kernel.h` API macro that changes the qualifiers on device functions?

Reply:
added more explanation: "Note that QUALIFIERS defined in curand_kernel.h is only for device functions and re-definition of the macro with the host extension is a limited scope for the random module of celeritas."